x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/settings_flyout/index.test.tsx

```diff
@@ -128,4 +137,34 @@ describe('SettingsFlyout', () => {
       expect(defaultProps.onClose).toHaveBeenCalled();
     });
   });
+
+  describe('when `assistantAttackDiscoverySchedulingEnabled` feature flag is enabled', () => {
+    beforeEach(() => {
+      (useIsExperimentalFeatureEnabled as jest.Mock).mockReturnValue(true);
+      render(
+        <TestProviders>
+          <SettingsFlyout {...defaultProps} />
+        </TestProviders>
+      );
+    });
+
+    it('should render Settings and Schedule tabs', () => {
+      expect(screen.getByRole('tab', { name: 'Settings' })).toBeInTheDocument();
+      expect(screen.getByRole('tab', { name: 'Schedule' })).toBeInTheDocument();
+    });
+
+    it('should render Settings and Schedule tabs 2', async () => {
+      expect(screen.getByRole('tab', { name: 'Settings' })).toBeInTheDocument();
+      expect(screen.getByRole('tab', { name: 'Schedule' })).toBeInTheDocument();
```

Comment:
Consider renaming `should render Settings and Schedule tabs 2` to something like `should render the schedule tab when clicked`, and eliminating the leading `expects` if they are covered by the previous test.